src/main/resources/rollback/V1_basic_tables_rollback.sql

```diff
@@ -0,0 +1,5 @@
+DROP TABLE messaging.chat_room_members;
```

Comment:
Let's rename the file following the flyway naming convention https://flywaydb.org/documentation/concepts/migrations.html#naming

So, in our case it would be `U1_basic_tables.sql`

[Optional] Also, I propouse to start a description from the verb to make it more reasonable.
For example, it might look like this - `U1_add_basic_tables.sql`

Reply:
Ok, good idea about verb at the start of description.
Corrected